PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+To streamline the process of creating and reviewing pull requests, we have created a template.
+It is not required to follow this template perfectly, but we encourage you to think about each header.
+Before you publish a pull request think about the definition of done for the issue you are trying to resolve.
+-->
+
+## Description
+<!--
+Provide a concise description on the feature/fix your pull request implements.
+-->
+
+## Changes
+<!--
+Provide a list of important changes.
+
+e.g.
+* Added tests
+* Renamed variable
+* Added topic
+-->
+
+## Issues
+<!--
+Provide keywords to the Jira issues.
+
+e.g. closes PM-34
```

Comment:
I would turn this around - move "Close PM-" out of the comments and put in on top, remove ##Issues and add a small comment description (e.g. "
<!--
Reference keyword to Jira issue
-->
")